### DISCUSSION TITLE
NPT-MD simulation under the periodic boundary conditions

### DISCUSSION THREAD

**Discussion by Yuta-Yoshimoto** [mir-group/allegro#57] [Q&A]
Dear Allegro developers,

We have tried running **NPT-MD** simulations of organic molecules under the **periodic boundary conditions**.
We trained our Allegro model on the ANI-2x dataset, which includes quantum chemical calculation data of small organic molecules composed of H, C, N, O, F, Cl, and S atoms in a vacuum state (i.e., **non-periodic boundary**).
During the training, we included the `StressForceOutput` option and trained the model using only force values.
Then, the force MAE on the test set was about 0.73 kcal/mol/angstrom, which seems to be sufficiently small.

Using the `pair_allegro` package and the potential file extracted from the trained model, we performed **NPT-MD** simulations of 100 2-chloroethanol molecules (300 K, 1 atm) using `pair_style allego3232` in LAMMPS.
Then, we found that the system density continued to decrease, exhibiting a very low value below 0.1 g/cm^3.

I am wondering whether the Allegro model with the `StressForceOutput` option trained on the forces of isolated molecules under the **non-periodic boundary conditions** can be applied to NPT-MD simulations of condensed systems under the **periodic boundary conditions**.

We used the following packages:
NequIP: main branch (latest, v0.5.6)
Allegro: main branch (latest)
pair_allegro: multicut branch

I would really appreciate it if you could give us some comments on this issue.

Best,
Yuta

**Comment by Linux-cpp-lisp** [comment]
Discussing by email
